src/icons/fileIcons.ts

```diff
@@ -1865,6 +1865,10 @@ export const fileIcons: FileIcons = {
         'vite.config.mts',
       ],
     },
+    {
+      name: 'vitest',
+      fileNames: ['vitest.config.js', 'vitest.config.mjs', 'vitest.config.ts'],
```

Comment:
```suggestion
      fileNames: [
        'vitest.config.ts',
        'vitest.config.mts',
        'vitest.config.cts',
        'vitest.config.js',
        'vitest.config.mjs',
        'vitest.config.cjs',
      ],
```
Ref this: https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/constants.ts#L16-L21